docs/quickstart/quickstart_chains/ethereum-uniswap.md

```diff
@@ -0,0 +1,264 @@
+# Ethereum Quick Start - Uniswap (Medium)
+
+## Goals
+
+This project serves as an excellent foundation for initiating your Ethereum SubQuery project. It meticulously indexes various Uniswap entities, including Swaps, Pool operations, Price ticks, Positions, and additionally computes statistical metrics.
+
+::: warning
+We suggest starting with the [Ethereum Gravatar example](./ethereum-gravatar). The Uniswap project is a lot more complicated and introduces some more advanced concepts.
+:::
+
+Now, let's move forward and fork the example code for this project from [here](https://github.com/subquery/ethereum-subql-starter/tree/main/Ethereum/ethereum-uniswap-v3)
```

Comment:
@bgodlin Why not to use the approach of initialising a boilerplate project and then updating all the files step by step? And only sharing the final code here not asking to fork final version of the project.

Reply:
I contemplated starting a boilerplate project and creating a step-by-step guide for updating files, but I ultimately chose not to proceed with it. My decision was based on the desire to maintain consistency with the format used in all other published quickstart guides. Now, the question is whether we should introduce a new format for this specific page and subsequently update the format for all other guides, or should we stick with the proposed structure for this article and then implement a new format for all quickstart guides in a single comprehensive pull request?

Reply:
But other published starter guides follow exactly this approach (starter boilerplate + changes to the project with a step by step guide)...?